### PR TITLE
[Refactor] Avoid exposing analytor's private members

### PR DESCRIPTION
**Pull request by liuyehcf**
## What type of PR is this：
- [ ] bug
- [ ] feature
- [ ] enhancement
- [ ] refactor
- [x] others

## Which issues of this PR fixes ：

Fixes #

## Problem Summary(Required) ：

Keep fields of `Analytor` private as far as possible, make the codebase more readable and clean

### CODE REVIEW

**Review by murphyatwork — DISMISSED** [review]

File: be/src/exec/vectorized/analytor.cpp [hunk 1]
```diff
@@ -564,16 +596,20 @@ void Analytor::remove_unused_buffer_values(RuntimeState* state) {
     }
 
     int64_t remove_count = remove_end_position - _removed_from_buffer_rows;
-    for (size_t i = 0; i < _agg_fn_ctxs.size(); i++) {
-        for (size_t j = 0; j < _agg_expr_ctxs[i].size(); j++) {
-            _agg_intput_columns[i][j]->remove_first_n_values(remove_count);
+
+    {
+        SCOPED_TIMER(_column_append_timer);
```
Comment (trueeyu) [inline be/src/exec/vectorized/analytor.cpp:601]: The name is ambiguous

Comment (liuyehcf) [reply be/src/exec/vectorized/analytor.cpp]: changed to `_column_resize_timer`

**Comment by liuyehcf** [comment]
run starrocks_clang-format

**Comment by liuyehcf** [comment]
run starrocks_clang-format

**Comment by wanpengfei-git** [comment]
run starrocks_clang-format

**Review by murphyatwork — APPROVED** [review]

**Review by trueeyu — APPROVED** [review]

**Pull request merged**